build/dependencies.props

```diff
@@ -15,10 +14,19 @@
     <SystemRuntimeSerializationPrimitivesVersion>4.3.0</SystemRuntimeSerializationPrimitivesVersion>
     <SystemRuntimeSerializationXmlVersion>4.3.0</SystemRuntimeSerializationXmlVersion>
     <SystemSecurityClaimsVersion>4.3.0</SystemSecurityClaimsVersion>
-    <SystemSecurityCryptographyCngVersion>4.5.0</SystemSecurityCryptographyCngVersion>
     <SystemTextEncoding>4.3.0</SystemTextEncoding>
     <SystemTextJson>4.7.2</SystemTextJson>
     <SystemTextEncodingsWeb>4.7.2</SystemTextEncodingsWeb>
     <SystemXmlXmlDocumentVersion>4.3.0</SystemXmlXmlDocumentVersion>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <SystemSecurityCryptographyCngVersion>6.0.0-preview.4.21253.7</SystemSecurityCryptographyCngVersion>
```

Comment:
>-preview.4.21253.7 [](http://example.com/codeflow?start=47&length=18)

This there a non-preview which works w/.NET 8?

Reply:
Looks like 5.0.0 works. Use that instead?

Reply:
No, all the 6s are preview right now

Reply:
Let's ship with the latest stable GA version.